src/core/Directus/Filesystem/Files.php

```diff
@@ -195,7 +195,7 @@ protected function getImageFromURL($url)
 
         $contentType = $this->getMimeTypeFromContentType($contentType);
 
-        if (strpos($contentType, 'image/') === false) {
+        if (strpos($contentType, 'image/') === false && strpos($contentType, 'pdf') === false) {
```

Comment:
Can we have an array of supported file type?

Create a sperate function which returns all the supported filetypes. 
Easy to extends.